contrib/artifactory/src/mill/contrib/ArtifactoryPublishModule.scala

```diff
@@ -0,0 +1,52 @@
+package mill.contrib.artifactory
+
+import artifactory.ArtifactoryPublisher
+import mill._, scalalib._, define.ExternalModule, publish.Artifact
+
+trait ArtifactoryPublishModule extends PublishModule {
+  def artifactoryUri: String
+
+  def artifactorySnapshotUri: String
+
+  def publishArtifactory(credentials: String,
+                         readTimeout: Int = 60000,
+                         connectTimeout: Int = 5000): define.Command[Unit] = T.command {
```

Comment:
Thinking out loud. If you would add the `artifactoryUri` and `artifactorySnapshoUri` as optional parameters, it would be easy to deploy to different repos, e.g. test-deployments without touching the `build.sc`.

Reply:
The ExternalModule has a task that lets you do that, though. Think it’s necessary to add it here too?

Reply:
It's not "necessary", just convenient, in case one needs to release selected modules. The `publishAll` target has way to select specific modules/artifacts.

Reply:
Fair point, added it now. :)

Reply:
Thanks!